samcli/local/lambda_service/service.py

```diff
@@ -0,0 +1,103 @@
+"""Local Lambda Service"""
+
+import logging
+import os
+
+from flask import Flask, jsonify
+
+LOG = logging.getLogger(__name__)
+
+
+class LocalLambdaService(object):
+
+    _DEFAULT_PORT = 3001
+    _DEFAULT_HOST = '127.0.0.1'
+
+    def __init__(self, function_name_list, lambda_runner, port=None, host=None, stderr=None):
+        """
+        Creates a Local Lambda Service
+
+        Parameters
+        ----------
+        function_name_list list of str
```

Comment:
I had two options here:

1. Use a list of Function Logical Ids
2. A list of Function NamedTuple

I went with the first to not couple them together but I think it might be ok. Wanted to get your thoughts on this @sanathkr

Reply:
This looks good. The NamedTuple has too much information which is not a concern of this class

Reply:
Ya that is why I originally leaned this way but couldn't completely convince myself it was right. :)